main.py

```diff
@@ -473,16 +474,19 @@ def run(self) -> None:
         self.telegram.updater.start_polling()
         self.telegram.updater.idle()
 
-
-CONFIG_FILE = "config.yaml"
+env_config_file = os.getenv('HUBIBOT_CONFIG_FILE')
```

Comment:
what about this version of code? (I am not discussing semantic)

```python
env_config_file  = os.getenv("HUBIBOT_CONFIG_FILE", Path(__file__).with_name("config.yaml"))
```

Reply:
Yeah that's clearly better, I had forgotten that os.getenv lets you specify a default. I'll update the pull request a bit later today and send it back your way.